posts/five-interns-walk-into-an-office.md

```diff
@@ -0,0 +1,44 @@
+
+## The Technology
+During our time at Ippon, we were exposed to dozens of exciting technologies that are commonplace in the modern development world. We wanted to quickly touch on some of these technologies (specifically the ones used during the Insurance Chatbot project) and our thoughts on them from an intern’s point of view. The technologies we worked with can be split into three broad categories, web development, DevOps, and Amazon Web Services.
+
+Jhipster was one of the exciting web development technologies that we were able to work with. The powerful application generator gave us a great jumping-off point for learning all the necessary technologies for building a modern web application. Lots of time was spent playing with the generator and exploring what it created. By learning the technologies through JHipster generated code, we were able to see the coding practices in use in a real-world application, an opportunity that is usually not commonplace in our college classes. Since most of us had little to no experience with these things previously, Jhipster had a bit of a steep learning curve for us. But given enough time to explore and accidentally break a few applications, we saw the huge benefits of using it.
+
+Through Jhipster we were exposed to Angular and Spring Boot among several other technologies. As the integral parts of the JHipster technology stack, we were able to gain valuable experience with the popular front-end and back-end platforms respectively. We were also exposed to the concept of microservices in application development, a very common technique among modern, successful software companies such as Amazon, Netflix, and Twitter. While this added a layer of complication to the development process, we believe it was well-worth the extra effort to learn because it will very likely come up again during our future careers.
+
+On the DevOps side of things, Jenkins made deploying our application on AWS much easier. The automation server gave many of us our first experience with DevOps, something that is often lacking from college computer science courses but is vitally important in the real world of software development. We created jenkins scripts to automatically test, deploy, and run our application any time someone pushed an update to the project’s GitHub repository.
+
+This project also utilized many different parts of the Amazon Web Services (AWS) platform. The Jenkins server was hosted on its own Amazon Elastic Compute Cloud (EC2) instance. Amazon’s elastic load balancer service allowed us to create a single entry point that would alternate between two identical EC2 instances hosting the application, making sure that if there was any issue with either instance, the user would never know and the application would stay up. Amazon’s relational database service (RDS) held all of our application’s data.
```

Comment:
May want to capitalize Elastic Load Balancer since it's a named product.

Reply:
+1, and above as well